pinot-tools/src/main/java/org/apache/pinot/tools/service/PinotServiceManager.java

```diff
@@ -210,18 +210,24 @@ public boolean stopPinotInstance(ServiceStartable instance) {
     }
   }
 
-  public void start() {
-    LOGGER.info("Registering service status handler");
-    ServiceStatus.setServiceStatusCallback(_instanceId, new PinotServiceManagerStatusCallback(this));
-
+  public void start(boolean bootstrapMode) {
     if (_port < 0) {
```

Comment:
Should this be `<=`?

Reply:
doens't really matter b/c it will not be set to a port that's non-zero in constructor.